VocaDbWeb/Web.config

```diff
@@ -41,6 +41,8 @@
   <system.web>
     <compilation debug="true" targetFramework="4.8" batch="false">
       <assemblies>
+        <!-- https://github.com/dotnet/standard/issues/542 -->
+        <add assembly="netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />

```

Comment:
Did you get the error? I was under the impression this should be unnecessary if AutoGenerateBindingRedirects and PackageReference are enabled.

Reply:
You're right. Reverted Web.config.